src/build.c

```diff
@@ -2256,7 +2256,7 @@ static void build_load_menu_grp(GKeyFile *config, GeanyBuildCommand **dst, gint
 	gsize prefixlen; /* NOTE prefixlen used in macros above */
 	GeanyBuildCommand *dstcmd;
 	gchar *key;
-	static gchar cmdbuf[3] = "  ";
+	static gchar cmdbuf[4] = "  ";
```

Comment:
I would simply remove the lengths specification.

Reply:
Then the initialisation would need to be extended, it seems GCC is convinced that a unsigned between 0-99 can be three bytes long?????

Reply:
Personally I would make it not static or comment on why it is static, and remove the initialization assignment which gets over-written below anyway, and probably just make it big enough to hold any old integer.

But for this PR I just did the simplest thing to work-around the warning.

Reply:
Seems to originate here: https://github.com/geany/geany/commit/1726e570921faa941919a4e07f1d6d3ec55d8bb6#diff-8ae5a6e4e9368d91c0c59edbde1627c2R1762

Reply:
Oh, i counted the spaces wrong. I thought it was changed to 4 to match the length of the string literal.